quint/src/parsing/parseErrors.ts

```diff
@@ -25,3 +25,19 @@ export function tooManySpreadsError(id: bigint): QuintError {
     data: {},
   }
 }
+
+export function undeclaredTypeParamsError(id: bigint, unboundTypeVariables: string[]): QuintError {
+  return {
+    code: 'QNT014',
+    message: `the type variables ${unboundTypeVariables.join(', ')} are unbound.
```

Comment:
Nice error message! Two thoughts for the future:
1. We should probably have a "details" or "explanation" field in the errors to include this second part of this error. I'd also like that for details of type unification errors.
2. This seems like a case where we'll be able to include a `fix` to the error data, and have LSP provide it as a code action, like we do for mode errors.